pypdf/_page.py

```diff
@@ -852,7 +852,7 @@ def _add_transformation_matrix(
                     FloatObject(e),
                     FloatObject(f),
                 ],
-                " cm",
+                b"cm",
```

Comment:
What is about the previous leading whitespace?

Reply:
the leading  is space is useless as writing to stream will add spaces between operators/operands. This is most probably an old typo